tests/pykafka/test_balancedconsumer.py

```diff
@@ -39,7 +39,7 @@ def setUpClass(cls):
     def test_consume_returns(self):
         """Ensure that consume() returns in the amount of time it's supposed to
         """
-        self._mock_consumer._setup_internal_consumer(start=False)
+        self._mock_consumer._setup_internal_consumer([], start=False)
```

Comment:
I half think that this `[]` should be a kwarg (as `None`, of course) to avoid code like this, but maybe it doesn't matter since it's private anyway.